apps/emqx_schema_registry/src/emqx_schema_registry_serde.erl

```diff
@@ -481,3 +501,135 @@ handle_schema_encode_and_tag(OurSchemaName, RegistryName, Data, Args) ->
             #{tag => true}
         )
     end.
+
+create_external_http_resource(Name, Params) ->
+    %% It's fine to generate a node-local unique name here: this will be stored only in
+    %% local ETS serde table.
+    ResName = <<"schema_registry:external_http:", Name/binary>>,
+    ResourceId = emqx_resource:generate_id(ResName),
```

Comment:
is it really necessary to use generated ID here?

Reply:
I guess not really, since schema name is already guaranteed to be unique.  :thinking: